jsk_calibration/euslisp/calibration.l

```diff
@@ -354,8 +368,7 @@
     nil)
   (:check-collision ()
     (let ((result (send robot-model :self-collision-check :pairs collision-links)))
-      (pprint (list 'result result))
-      (if result (return-from :check-collision result))
+      (unless result (return-from :check-collision result))
```

Comment:
Is this correct?
~~~
result -> nil (not collide) or collision pair (collide)
 if result -> return collision-pair
 unless result -> check robust collision
~~~